packages/grid/src/styled/index.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { IWrapper } from '../interfaces';
+import { getWrapperStyles } from './utils';
+
+
+export * from './body';
+export * from './header';
+export * from './totals';
+
+
+export const Wrapper = styled.div<IWrapper>`
+  width: ${({ width }) => `${width}px`};
+  overflow: hidden;
+  border-radius: 10px;
+  ${({ changingColumns }) => {
+    if (changingColumns === 'move') {
+      return `
+        -webkit-touch-callout: none;
```

Comment:
It is unnecessary to add vendor prefixes manually. Styled components will do it for you.